src/group-constructors.js

```diff
@@ -19,7 +19,11 @@
   }
 
   function constructor(children, timingInput) {
+    this._parent = null;
     this.children = children || [];
+    for (var i = 0; i < this.children.length; i++) {
+      this.children[i]._parent = this;
```

Comment:
> Does re-parenting happen somewhere? (removing from a former _parent)

No, and there is a FIXME for this somewhere. I should probably add that to this CL also.